### PR TITLE
LZA-21: Add detail to Strategic first

### PR DESCRIPTION
**Pull request by nickmcmahon01**

### CODE REVIEW

File: documentation/principles/strategic-first.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+order: 104
+title: Be Strategic first
+excerpt: We will meet requirements in a strategic way
+tags:
+- principle
+eleventyNavigation:
+  parent: principles
+---
+
+**Our approach is to be strategic first.** \
```
Comment (LiamMacP) [inline documentation/principles/strategic-first.md:12]:
```suggestion
**Our approach is to be strategic first.**
```

File: documentation/principles/strategic-first.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+Where new requirements are identified we will try and meet them in a strategic way.
```
Comment (LiamMacP) [inline documentation/principles/strategic-first.md:13]:
```suggestion
Where new requirements are identified, we will try and meet them in a strategic way.
```

File: documentation/principles/strategic-first.md [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+
+This means we will make informed, purpose-driven choices which increase our chances of achieving our long-term desired outcomes.
+
+We will defer to the Home Office strategic solutions by default then favour AWS native tooling and managed services over custom tooling.
```
Comment (LiamMacP) [inline documentation/principles/strategic-first.md:17]:
```suggestion
We will defer to the Home Office strategic solutions then favour AWS native tooling and managed services over custom tooling.
```

Not sure by default is adding anything there?

**Review by LiamMacP — APPROVED** [review]
Fine for now, can't think of anything off the top of my head that should be added. I have put a few suggestions in...

**Comment by nickmcmahon01** [comment]
> I don't think the principal should be changed from being native first. I just think it need to acknowledge that strategy takes precedence. E.g. "Where no strategic decision exists to govern our choice, our approach is to be native first ...."

In that case I wonder if there's two, one is native first, and one is 'solve for all'

File: documentation/principles/native-first.md [hunk 4]
```diff
@@ -1,13 +1,15 @@
 ---
 layout: sub-navigation
 order: 104
-title: Native first
-excerpt: Short extract
+title: Be native first
+excerpt: We will meet requirements in a strategic way
```
Comment (LiamMacP) [inline documentation/principles/native-first.md:5]:
```suggestion
excerpt: We will meet requirements in a native way
```

**Review by stephenkaye-ho — APPROVED** [review]

**Review by LiamMacP — APPROVED** [review]

**Pull request merged**